### ISSUE TITLE
在HRSC2016数据集上，用您的代码训练36轮后测试的精度是89.06(RTX3090, samples_per_gpu=8)，并没有达到90.41，请问是什么原因造成的，可以提供一下训练好的checkpoint吗

### ISSUE THREAD

**Issue by panyushan-cn** [LeapLabTHU/ARC#4]

**Comment by yifanpu001** [comment]
Hi, we set the hyper-parameter `samples_per_gpu `to 2, the same as the baseline model Oriented-RCNN. I assume you are training your model with a single GPU, so the batch size is 2. Generally, you increase the batch size by four times while keeping the learning rate the same (I assume). This is likely to cause a performance degradation

Thank you for your suggestion. We now plan to clean up and release the code and checkpoints for the HRSC dataset. However, due to the upcoming CVPR deadline, you may need to wait for a few days.

**Comment by panyushan-cn** [comment]
> Hi, we set the hyper-parameter `samples_per_gpu `to 2, the same as the baseline model Oriented-RCNN. I assume you are training your model with a single GPU, so the batch size is 2. Generally, you increase the batch size by four times while keeping the learning rate the same (I assume). This is likely to cause a performance degradation
> 
> Thank you for your suggestion. We now plan to clean up and release the code and checkpoints for the HRSC dataset. However, due to the upcoming CVPR deadline, you may need to wait for a few days.

Ok，thanks
